lib/elixir/lib/calendar.ex

```diff
@@ -90,6 +90,28 @@ defmodule Calendar do
           microsecond: microsecond
         }
 
+  @typedoc """
+  Specifies the time zone database for calendar operations.
+
+  Many functions in the `DateTime` module requires a time zone database.
+  By default, it uses the default time zone database returned by
+  `Calendar.get_time_zone_database/1`, which defaults to
+  `Calendar.UTCOnlyTimeZoneDatabase` which only handles "Etc/UTC"
+  datetimes and returns `{:error, :utc_only_time_zone_database}`
+  for any other time zone.
+
+  Another time zone database, such as one originating from a package,
+  can be passed as argument or set globally, either via configuration:
```

Comment:
I think this sentence is a bit awkward. I had to read it twice to understand it. Maybe something like this would be better?

```
Other time zone databases (including ones provided by packages) can be configured
as default either via configuration:

    config :elixir, :time_zone_database, CustomTimeZoneDatabase

or by calling `Calendar.put_time_zone_database/1`
```